README.md

```diff
@@ -92,9 +129,11 @@ Is injective. In a diagram, the arrow is often written with a hook (as in $\hook
 \subsubsection*{Möbius band}
 \subsubsection*{Mapping cylinder}
 
-\subsubsection*{Neighborhood}
+\subsubsection*{Neighborhood} A Neighborhood of a point $x$, is just an open set
+containing that point $x$ under the given Topology.
+
 \subsubsection*{Null-homotopic}
-A function is null-homotopy if it's at an end of a null-homotopy.
+A function is null-homotopic if it's at an end of a null-homotopy.
```

Comment:
Thinking about it, I think this should be `a space`, not `a function` 
*thinkingsmiley*

Reply:
No, it's a function.  A function f is null-homotopic if it is homotopic to a constant map.
A space is contractible if the identity specifically is null-homotopic.